CONTRIBUTING.rst

```diff
@@ -15,10 +15,12 @@ Reporting Bugs
 to produce an incorrect or unexpected result, or to behave in unintended
 ways.
 
-Bugs are tracked as `JIRA
-issues <https://jira.hyperledger.org/projects/IR/issues/IR-275?filter=allopenissues&orderby=issuetype+ASC%2C+priority+DESC%2C+updated+DESC>`__
+Bugs are tracked as `GitHub issues <https://github.com/hyperledger/iroha/issues>` (this is the preferred option) or as `JIRA
+issues <https://jira.hyperledger.org/projects/IR/issues/IR-275?filter=allopenissues&orderby=issuetype+ASC%2C+priority+DESC%2C+updated+DESC>` (if it is convenient to you)__
```

Comment:
Let's use centralized the only one place on GitHub.

Reply:
Let's not reject HL Jira completely for now, many people are not aware about gihub issues thing still